README.md

```diff
@@ -703,6 +704,24 @@ Stellar.Horizon.PaymentPaths.list_send_paths(source_asset_type: :native, source_
 
 See [**Stellar.Horizon.Paths**](https://developers.stellar.org/api/aggregations/paths/) for more details.
 
+### Order Books
+
+#### Retrieve order Books
+Provides an order book’s bids and asks:
+- [Required] selling_asset_type. The type for the asset being sold (base asset), **native**, **credit_alphanum4**, or **credit_alphanum12**.
+- [Required] buying_asset_type. The type for the asset being bought (counter asset), **native**, **credit_alphanum4**, or **credit_alphanum12**.
+- [Optional] selling_asset_issuer. The Stellar address of the issuer of the asset being sold (base asset). Required if the selling_asset_type is not native
+- [Optional] selling_asset_code. The code for the asset being sold (base asset). Required if the selling_asset_type is not native.
+- [Optional] buying_asset_issuer. The Stellar address of the issuer of the asset being bought (counter asset). Required if the **buying_asset_type** is not **native**.
+- [Optional] buying_asset_code. The code for the asset being bought (counter asset). Required if the **buying_asset_type** is not **native**.
+- [Optional] limit. The maximum number of records returned
+
+```elixir
+Stellar.Horizon.OrderBooks.retrieve(selling_asset_type: :native, buying_asset_type: :native)
+```
```

Comment:
@MrJuanGaviriaK to make the endpoints easy to use, we definitely can improve the definition for this endpoint interface.

**My proposal:**

```elixir
Stellar.Horizon.OrderBooks.retrieve(selling_asset: :native, buying_asset: :native, limit: 20)
Stellar.Horizon.OrderBooks.retrieve(selling_asset: :native, buying_asset: :native, limit: 10)
...
Stellar.Horizon.OrderBooks.retrieve(selling_asset: [code: "BTC", issuer: "ISSUER"], buying_asset: :native, limit: 10)
```

Internally, you can resolve the required params for the endpoints, and in this way, provide a better UX.

What do you think?

Reply:
I agree, I'm going to improve it, thanks!